src/dss/main.py

```diff
@@ -65,7 +65,15 @@ def initialize_command(kubeconfig: str) -> None:
     "--kubeconfig",
     help=f"Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to '{KUBECONFIG_DEFAULT}'.",  # noqa E501
 )
-def create_notebook_command(name: str, image: str, kubeconfig: str) -> None:
+@click.option("--no-gpu", is_flag=True, help="Create a notebook without GPU support.")
```

Comment:
Why do we need a `--no-gpu`?  Can the absence of `--gpu=something` imply no gpu, and we get rid of this?

Reply:
Its recommendation of ux team to provide this `--no-gpu` flag

Reply:
I wonder if there's a miscommunication here.  If I understand correctly, these two commands are both the same?

```
dss create my-notebook
dss create my-notebook --no-gpu
```

Am I missing something about the CLI?